CHANGELOG.md

```diff
@@ -2,10 +2,33 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.1 June 10, 2021
+# 10.0.0-rc.2 June 23, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Features ✨ and improvements 🏁
+* Add Persistent Layer API so that the tagged layer and its associated resources would remain when a style is reloaded. This improves performance of Annotation and Location Component Plugin during the style change. ([#368](https://github.com/mapbox/mapbox-maps-android/pull/368))
+* Add activity showcasing style cross fading. ([#436](https://github.com/mapbox/mapbox-maps-android/pull/436))
```

Comment:
We don't need to add example changes

Reply:
we could have an example section, but agree that this doesn't belong to features.

Reply:
This one is more of a feature requested. Although it's not part of an SDK I'd leave it here to make it more discoverable for SDK users.

Reply:
I think as long as it's not a feature landed in the sdk, we shouldn't put it in the features section.